stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/querychannels/QueryChannelsControllerIntegrationTest.kt

```diff
@@ -160,50 +157,6 @@ internal class QueryChannelsControllerIntegrationTest : BaseConnectedMockedTest(
             sut.channels.value shouldBeEqualTo listOf(data.channel1, data.channel2, data.channel3)
         }
 
-    @Test
-    fun `Given channel in the list and default events handler When handle NotificationMessageNewEvent Should not make request to API`(): Unit =
```

Comment:
I wouldn't just delete these tests. If logic gets changed we should reflect it in our test. Now, `DefaultChatEventHandler` adds a channel from this event if it's absent what about checking it? If you has questions how to write such tests I can assist you

Reply:
I added unit tests in the new classes that cover the new cases added